ui/src/common/containers/__tests__/__snapshots__/EditRecordActionContainer.test.jsx.snap

```diff
@@ -3,69 +3,77 @@
 exports[`EditRecordActionContainer does not render edit button if user is not cataloger or superuser 1`] = `""`;
 
 exports[`EditRecordActionContainer renders edit button if user is cataloger 1`] = `
```

Comment:
this isn't cool, should be a shallow render rather than `mount`. But I see that it's something PR has introduced, we can have a look together maybe after you're done with everything else.